tests/NHSD.BuyingCatalogue.Solutions.API.UnitTests/SolutionsControllerGetPublicTests.cs

```diff
@@ -20,51 +22,37 @@ namespace NHSD.BuyingCatalogue.Solutions.API.UnitTests
     [TestFixture]
     public sealed class SolutionsControllerGetPublicTests
     {
-        private Mock<IMediator> _mockMediator;
-
-        private SolutionsController _solutionsController;
-
-        private const string SolutionId1 = "Sln1";
-        private const string SolutionId2 = "Sln2";
-
-        private readonly DateTime _lastUpdated = DateTime.Today;
-
         [SetUp]
         public void Setup()
         {
             _mockMediator = new Mock<IMediator>();
             _solutionsController = new SolutionsController(_mockMediator.Object);
+            _claimedCapabilityList = new List<IClaimedCapability>();
```

Comment:
This section will be run for every test, does every test need a list of claimed capabilities?